src/test/resources/mssql/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+if ! [ -f /var/opt/mssql/.initialized ]; then
+  while ! </dev/tcp/localhost/1433 2>/dev/null; do
+    sleep 2
+  done
+  # Just waiting a little to be sure.
+  sleep 5
```

Comment:
I am not liking this, we'll have flaky tests.

Reply:
I don't think there's any need to have a custom entry point for the DB. Instead we can create the testcontainer normally and then call `org.testcontainers.ext.ScriptUtils.runInitScript(...)` after calling `JdbcDatabaseContainer#start` in  order to execute `.setup.sql`.

We take this approach in the cache-manager:

https://github.com/gingersnap-project/cache-manager/blob/354fd06c2b2a7f2b3a4c0642d30fe5d8490b0a0a/manager/src/test/java/io/gingersnapproject/mssql/MSSQLServerResources.java#L17-L29

Reply:
Thanks a lot!! I updated to use the `withInitScript`, let's see how it goes with CI.